src/services/completions.ts

```diff
@@ -693,6 +696,11 @@ export function getCompletionsAtPosition(
     switch (completionData.kind) {
         case CompletionDataKind.Data:
             const response = completionInfoFromData(sourceFile, host, program, compilerOptions, log, completionData, preferences, formatContext, position);
+            if (response && !includeApiData) {
+                for (const entry of response.entries) {
+                    delete entry.symbol;
+                }
+            }

```

Comment:
The logic needs to be structured in such a way that the default case (when `includeApiData` is false) does no extra work.